### PR TITLE
Build both projs

### PR DESCRIPTION
**Pull request by ocefpaf**
@xylar let's see if this works!

### CODE REVIEW

**Review by xylar — APPROVED** [review]
As long as the tests pass, this looks great to me.

File: recipe/conda_build_config.yaml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+proj:
+    - '7.2.0'
```
Comment (snowman2) [inline recipe/conda_build_config.yaml:2]: `7.2.0` or `7.2.1`?

Comment (ocefpaf) [reply recipe/conda_build_config.yaml:2]: Good question. I went with the one from the [previous pin](https://github.com/conda-forge/conda-forge-pinning-feedstock/blob/master/recipe/conda_build_config.yaml#L602). I guess we skipped 7.2.1. For now I would keep 7.2.0 to maintain compatibility with the rest of the stack.

Comment (snowman2) [reply recipe/conda_build_config.yaml:2]: Makes sense 👍

Comment (xylar) [reply recipe/conda_build_config.yaml:2]: Yes, I agree.

**Comment by snowman2** [comment]
Thanks @ocefpaf :+1:

**Pull request merged**